dir.props

```diff
@@ -28,6 +28,7 @@
 
   <!-- list of nuget package sources passed to dotnet -->
   <ItemGroup Condition="'$(ExcludeInternetFeeds)' != 'true'">
+    <DotnetSourceList Include="$(OverridePackageSource)" />  
     <DotnetSourceList Include="https:%2F%2Fdotnet.myget.org/F/dotnet-core/api/v3/index.json" />
```

Comment:
For consuming the blob feed I think all you need to do is add it the list here. 

This is of course a different scenario from actually being able to override it which we want also do for the orchestrated build so depending on your goal you might want or the other.

Reply:
For the later we will be using a new name PB_RestoreSource as the input feed.

Reply:
This may address my concern which you queried in the previous comment.

Reply:
In the build definition, are we calling the `OverridePackageSource` as `PB_AzureFeedUrl`, `PB_OverridePackageSource`, something else?

Reply:
I think you want 2 things:

1. Add the `https://dotnetfeed.blob.core.windows.net/dotnet-core/packages/index.json` to the feed list here.
2. In the build definition take an incoming PB_RestoreSource variable and pass it to the build. You can still use OverridePackageSource or whatever in the targets/props file in the repo.